### PR TITLE
Dynamic input file - input files for fc, nvme, nvmf and vfc.

### PR DESCRIPTION
**Pull request by manvanthar**
Added the cfg file changes required for the fc, nvme, nvmf and vfc.

### CODE REVIEW

File: config/inputs/io_nvme_input.txt [hunk 1]
```diff
@@ -1,26 +1,25 @@
 [io_nvme]
-device = "nvme0"
```
Comment (abdhaleegit) [inline config/inputs/io_nvme_input.txt:2]: we need device parameter to cover the 8 nvme functinal test in cfg

**Review by abdhaleegit — CHANGES_REQUESTED** [review]
few things
1. the io_nvme_fvt.cfg bucket you are refering to is old,  the latest updates are in io_nvme.cfg  the design expected us to create another cfg ending with _fvt .. so we need to update io_nvme_fvt.cfg to look like io_nvme.cfg   

2. with new bucket,  input file needs device parameter like device = nvme0  and disk= /dev/nvme0n1 

3. io_nvme.cfg will first run all the nvme functional tests and than creates nvme namespace devices for successive nvme stress test in same bucket .. at the end of bucket delete all nvme namespaces    

refer to https://github.ibm.com/ltctest/avocado-fvt-wrapper/blob/master/config/tests/host/io_nvme.cfg

File: config/inputs/io_nvme_input.txt (hunk 1 above)
Comment (manvanthar) [reply config/inputs/io_nvme_input.txt:2]: Added the changes specific to nvme file

**Comment by manvanthar** [comment]
Abdul thanks for the review.  Following changes are done

1. reverted back the "device" to nvme input file
2.  matched the nvme.cfg file as per the ltctest gitfolder

**Review by abdhaleegit — APPROVED** [review]
LGTM

File: config/inputs/io_nvme_input.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
 [io_nvme]
+device = "nvme0"
```
Comment (PraveenPenguin) [inline config/inputs/io_nvme_input.txt:2]: can this be added in first commit itself

Comment (manvanthar) [reply config/inputs/io_nvme_input.txt:2]: Done all the changes are now in one commit

**Review by PraveenPenguin — APPROVED** [review]

**Pull request merged**